stable/datadog/README.md

```diff
@@ -300,5 +316,13 @@ helm install --name <RELEASE_NAME> \
 | `clusterchecksDeployment.nodeSelector`                   | Node selectors                                                                                | `nil`                                       |
 | `clusterchecksDeployment.affinity`                       | Node affinities                                                                               | avoid running pods on the same node         |
 | `clusterchecksDeployment.livenessProbe`                  | Overrides the default liveness probe                                                          | http port 5555                              |
-| `clusterchecksDeployment.rbac.dedicated`                  | If true, use dedicated RBAC resources for clusterchecks agent's pods                          | `false`                                     |
-| `clusterchecksDeployment.rbac.serviceAccount`            | existing ServiceAccount to use (ignored if rbac.create=true) for clusterchecks                | `default`                                   |
\ No newline at end of file
+| `clusterchecksDeployment.rbac.dedicated`                 | If true, use dedicated RBAC resources for clusterchecks agent's pods                          | `false`                                     |
+| `clusterchecksDeployment.rbac.serviceAccount`            | existing ServiceAccount to use (ignored if rbac.create=true) for clusterchecks                | `default`                                   |
+| `systemProbe.enabled`                  | If true, enable system probe collection                                                                    | `false`|
+| `systemProbe.debugPort`                | The port to expose pprof and expvar for system-probe agent                                                 | `0`    |
+| `systemProbe.enableConntrack`          | If true, system-probe connects to the netlink/conntrack subsystem to add NAT information to connection data. Ref: http://conntrack-tools.netfilter.org/| `true`|
```

Comment:
Doesn't that require to mount debugfs ? If so, why defaulting to true ?

Reply:
No, this is only useful if system-probe is enabled, but when system-probe is enabled, we want this to be default to `true`.

Reply:
We might want to include 

> Also if for some reason the conntrack failed, system-probe won't quit, it'll just print a warning message and carry on.

In the docs